paper-slider.html

```diff
@@ -271,15 +266,15 @@
     },
 
     _minChanged: function() {
-      this.setAttribute('aria-valuemin', this.min);
+      this.$.sliderContainer.setAttribute('aria-valuemin', this.min);
```

Comment:
Does this actually fix the problem? Did you test it with a screen reader? Are there any notable differences between Shadow- and Shadey- DOM?